src/scripting/sector.hpp

```diff
@@ -50,6 +50,20 @@ class Sector final : public GameObjectManager
    * @param float $gravity
    */
   void set_gravity(float gravity);
+
+  /**
+ * Adds a game object to the game
+ * 
+ * @param class_name GameObject's class
+ * @param name Name of the created object
+ * @param posX X position inside the current sector
+ * @param posY Y position inside the current sector
+ * @param direction Direction 
+ * @param data Additional data
+ */
```

Comment:
A few issues with this comment block:

1. It could be more descriptive of the fact that only moving objects could be added with this command.
2. It should have full stops at the end of the main and parameter descriptions.
3. The parameter formatting is wrong. It should rather be `@param type $name description`.
4. Everything below the line with `/**` should be indented 2 spaces further.

Reply:
https://www.doxygen.nl/manual/docblocks.html